Source/Archiver.h

```diff
@@ -63,7 +63,7 @@ SOFTWARE
 
 #define	ARCHIVER_SETTINGS_FILE_PATH		"/boot/home/config/settings/"
 #define	ARCHIVER_SETTINGS_FILE			"archiver.settings"
-#define	ARCHIVER_RULES_FILE_PATH		"/boot/home/config/etc/"
+#define	ARCHIVER_RULES_FILE_PATH		"/boot/home/config/non-packaged/etc/"
```

Comment:
I think it is better to look in the packaged directory? Or does it try to write here?
Ideally we should look in both but it is out of the scope of your task